Public/Src/Cache/ContentStore/Distributed/Sessions/DistributedContentSession.cs

```diff
@@ -179,7 +179,9 @@ private Task<TResult> PerformPutFileGatedOperationAsync<TResult>(OperationContex
 
             if (putResult.Succeeded && Settings.EnableProactiveCopy)
             {
-                RequestProactiveCopyIfNeededAsync(context, putResult.ContentHash).FireAndForget(context);
+                // Since the rest of the operation is done asynchronously, create new context to stop cancelling operation prematurely.
+                var operationContext = new OperationContext(context.TracingContext);

```

Comment:
>new OperationContext(context.TracingContext) [](start = 39, length = 44)

Use OperationContext method from StartupShutdownSlimBase since it will cancel on shutdown.